.github/CONTRIBUTING.md

```diff
@@ -138,6 +138,10 @@ The previous code made compliant:
   code
 ```
 
+### User Interfaces
```

Comment:
That was voted on between other maintainers and has been deemed as the way forward.